src/FlightDisplay/GuidedAltitudeSlider.qml

```diff
@@ -24,10 +24,18 @@ Rectangle {
     property var  _activeVehicle:       QGroundControl.multiVehicleManager.activeVehicle
     property var  _flyViewSettings:     QGroundControl.settingsManager.flyViewSettings
     property real _vehicleAltitude:     _activeVehicle ? _activeVehicle.altitudeRelative.rawValue : 0
+    property real _vehicleAltitudeAMSL: _activeVehicle ? _activeVehicle.altitudeAMSL.rawValue : 0
     property bool _fixedWing:           _activeVehicle ? _activeVehicle.fixedWing : false
     property real _sliderMaxAlt:        _flyViewSettings ? _flyViewSettings.guidedMaximumAltitude.rawValue : 0
     property real _sliderMinAlt:        _flyViewSettings ? _flyViewSettings.guidedMinimumAltitude.rawValue : 0
     property bool _flying:              _activeVehicle ? _activeVehicle.flying : false
+    property bool _useAMSL:             _flyViewSettings ? _flyViewSettings.guidedAltitudeAMSL.rawValue : false
+    property bool _useDelta:            _flyViewSettings ? _flyViewSettings.guidedAltitudeDeltaMode.rawValue : false
+    property real _sliderMaxChange:     _flyViewSettings ? _flyViewSettings.guidedAltitudeMaxChange.rawValue : 0
+    property real _distanceToGround:    _activeVehicle ? _activeVehicle.distanceSensors.rotationPitch270.value : NaN
```

Comment:
Seems a little specific, probably want some concept of "distance to ground sensor" other than Pitch270. But this is probably fine as it is for us.

Reply:
Agree that is is not optimal, but it is at least loosely coupled to "down" in src/Vehicle/DistanceSensorFact.json